functions/imagemagick/index.js

```diff
@@ -103,6 +103,7 @@ const blurImage = async (file, blurredBucketName) => {
   }
 
   // Delete the temporary file.
-  return promisify(fs.unlink(tempLocalPath));
+  const unlink = promisify(fs.unlink);
+  return unlink(tempLocalPath);
```

Comment:
Please don't use the npm package`util's `promisify` since you can use the native Node `fsPromises.unlink`.

Remove
```js
const {promisify} = require('util');
```

Add
```js
const fsPromises = require('fs').promises;
```

https://nodejs.org/api/fs.html#fs_fspromises_unlink_path

Reply:
The docs say `fs.promises` is only available in Node 10+ - AFAIK these samples (still) need to support Node 8.

Reply:
Yes, this is why I didn't use it.

Reply:
Got it. Disregard the suggestion then.

Reply:
Yes, `fs.promises` is not supported in Node 8, and it's experimental in Node 10.